it/it-plugins/billing-plugin/src/main/java/BillingValidations.java

```diff
@@ -44,6 +44,14 @@ public void checkOnProjectAnalysis(Organization organization) {
   }
 
   @Override
+  public void checkCanUpdateProjectsVisibility(Organization organization, boolean updateToPrivate) {
```

Comment:
I've added this 's' because if we remove it, we should probably add a ComponentDto, otherwise it may be misleading...
WDYT ?